packages/client/src/base/focus/focus-tracker.ts

```diff
@@ -29,7 +30,7 @@ export interface FocusChange {
  * Allows querying of the current focus state and the focused root diagram element and the currently focused element within the diagram.
  */
 @injectable()
-export class FocusTracker implements IActionHandler {
+export class FocusTracker implements IActionHandler, IDiagramStartup {
     protected inActiveCssClass = 'inactive';
     protected _hasFocus = true;
```

Comment:
I wonder if that should really be true now that we actually track the focus change in the beginning?